src/services/comments.js

```diff
@@ -53,22 +53,22 @@ module.exports = class CommentsHelper {
 				return responses.successResponse({
 					statusCode: httpStatusCode.ok,
 					message: 'COMMENT_UPDATED_SUCCESSFULLY',
-					result: commentCreate,
+					result: {},
 				})
 			}
 
 			//update the comment
-			if (bodyData.status === common.STATUS_RESOLVED) {
-				bodyData.resolved_by = userId
-				bodyData.resolved_at = new Date()
+			if (bodyData.comment.status === common.STATUS_RESOLVED) {
+				bodyData.comment.resolved_by = userId
+				bodyData.comment.resolved_at = new Date()
 			}
 
 			const filter = {
 				resource_id: resourceId,
 				id: commentId,
 			}
 
-			const [updateCount, updatedComment] = await commentQueries.updateOne(filter, bodyData, {
+			const [updateCount, updatedComment] = await commentQueries.updateOne(filter, bodyData.comment, {
```

Comment:
@adithyadinesh0412  how the response message will change? on basis of what COMMENT_RESOLVED' : 'COMMENT_UPDATED_SUCCESSFULLY',

Reply:
as discussed , resolving